app/script/auth/module/reducer/authReducer.js

```diff
@@ -122,6 +123,12 @@ export default function reducer(state = initialState, action) {
     case AuthActionCreator.ENTER_PERSONAL_CREATION_FLOW: {
       return {...state, isInTeamFlow: false};
     }
+    case UserActionCreator.USER_SEND_ACTIVATION_CODE_SUCCESS: {
```

Comment:
In order to set `fetching` to false in an error case there should be an `USER_SEND_ACTIVATION_CODE_FAILED `